src/MensaWidget/MWFrame.class.st

```diff
@@ -140,3 +233,17 @@ MWFrame >> removeScalability [
 			[each class superclass = CornerGripMorph]) 
 				ifTrue: [each abandon]]
 ]
+
+{
+	#category : #creation,
+	#'squeak_changestamp' : 'Louis Christ 6/7/2024 20:55'
+}
+MWFrame >> showMealsFor: aDate [
+	
+	| contentMorph |
+	
+	contentMorph := self buildContentMorph.
+	(MWDataGettingWrapper getMealsOnDate: aDate) do: 
+		[:each | contentMorph addMorph: each getCanteenMorph].
+	self contentMorph scroller addMorph: contentMorph.
```

Comment:
as stated above regarding :each and the "."

Reply:
and no dot needed at the end

Reply:
i do not quite get what the problem here is since the dot was the comment above